hack/verify-tls-info.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+function cleanup() {
+    return_code=$?
+    os::test::junit::generate_report
+    os::util::describe_return_code "${return_code}"
+    exit "${return_code}"
+}
+trap "cleanup" EXIT
+
+os::test::junit::declare_suite_start "verify/generated"
+os::cmd::expect_success "${OS_ROOT}/hack/update-tls-info.sh"
```

Comment:
I don't speak bash.  Is this mutating files in my repo?  If so, we need a different approach that doesn't do that.  Generating to a tmp directory could work.

Reply:
Yes, similar to `verify-generated` - it updates the code and checks if any change appeared in `git diff`.

We know which files are generated, so I guess this script could use tempdir, yeah